From: Chris Mason <mason@suse.com>
Subject: Make proc_event_num_listeners __read_mostly

This will lower the fast path costs of the userland connector code.

Acked-by: Jeff Mahoney <jeffm@suse.com>

diff -r 61ab85753515 drivers/connector/cn_proc.c
--- a/drivers/connector/cn_proc.c	Tue Jan 24 14:14:58 2006 -0500
+++ b/drivers/connector/cn_proc.c	Tue Jan 24 14:32:23 2006 -0500
@@ -32,7 +32,7 @@
 
 #define CN_PROC_MSG_SIZE (sizeof(struct cn_msg) + sizeof(struct proc_event))
 
-static atomic_t proc_event_num_listeners = ATOMIC_INIT(0);
+static atomic_t proc_event_num_listeners __read_mostly = ATOMIC_INIT(0);
 static struct cb_id cn_proc_event_id = { CN_IDX_PROC, CN_VAL_PROC };
 
 /* proc_event_counts is used as the sequence number of the netlink message */
